client/core/bookie.go

```diff
@@ -591,14 +594,46 @@ func handleTradeResumptionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 	return nil
 }
 
-func (dc *dexConnection) refreshServerConfig() error {
+// refreshServerConfig fetches and replaces server configuration data. It also
+// initially checks that a server's API version is one of apiVers.
+func (dc *dexConnection) refreshServerConfig(apiVers []int) error {
```

Comment:
Why not use the package-level var `serverAPIVers` instead of an arg?

Reply:
For testing...
If we use the package level variable, it makes testing in parallel a bit difficult, although doable. Will see what it looks like, I guess will need to add a lock to the test harness.

Reply:
Ah, I see.  I don't think there will be any concurrency issues though. I think you can just change the slice and and call `refreshServerConfig` for each test case.  But if for some reason something is a problem with that, I don't think you even need a running tCore.  That is, you could manually put together a `dexConnection` either with `testDexConnection` or manually with just what's needed to test this method.

Reply:
Just using the top numbers of a uint16 which are probably not going to ever be actual versions.